### ISSUE TITLE
[Standalone] Bug when selecting uncommon input channels

### ISSUE THREAD

**Issue by elieserdejesus** [elieserdejesus/JamTaba#932]
An user reported a problem with your audio interface. Using the verison 2.1.0 he can select any input pairs (1-2, 3-4, 5-6, etc). But in the v2.1.1 the inputs 1-2 are working ok, but when he selected the inputs 3-4 (or another pair) JTBA crash. 

I tested in my Fast Track Ultra (6 inputs) and no crash, is working fine.

**Comment by elieserdejesus** [comment]
@jonjamcam , @pljones , @Hitrate , @DoublebassOO , @MagnusDrumsTrumpet , @qulf ? Can you help testing audio inputs in Standalone? 

The issue is related with uncommon input pais, like 3&4, 5&6, etc.

**Comment by elieserdejesus** [comment]
I found the problem, I programmed a bad code in the last version. I think the current JTBA version can't work with more than 2 channels. My bad, drink and programming is not work sometimes 😄 . 

The old code (v2.1.0 in left) and the new code in right side. The new code is not a correct replacement:
![image](https://user-images.githubusercontent.com/1012741/31588473-d178fe00-b1d0-11e7-88ae-db47f7b60f07.png)
